src/axom/mint/docs/sphinx/sections/getting_started.rst

```diff
@@ -0,0 +1,275 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _sections/getting_started:
+
+Getting Started with Mint
```

Comment:
Thanks for this "getting started" tutorial.  This is exactly what we need.

Reply:
Sure, glad you liked it.